web/src/hooks/mutations/useEditProjectMetaMutation.ts

```diff
@@ -12,7 +12,7 @@ type EditProjectMeta = {
   newIsPrivate?: boolean;
   newName?: string;
   newTag?: string;
-  newSchema?: string;
+  newSchema?: string | undefined;
```

Comment:
I dont think the `| undefined` is needed when you have `newSchema?:` right?

Reply:
fixed